DxDispatch/src/dxdispatch/dxDispatchWrapper.cpp

```diff
@@ -94,16 +94,9 @@ HRESULT DxDispatch::RuntimeClassInitialize(
         m_logger->LogError(fmt::format("Failed to parse command-line arguments: {}", e.what()).c_str());
         throw;
     }
-   
-
-    SetDisableAgilitySDK(m_options->DisableAgilitySDK());
-
-    // Needs to be constructed *before* D3D12 device. A warning is printed if DXCore.dll is loaded first,
-    // even though the D3D12Device isn't created yet, so we create the capture helper first to avoid this
-    // message.
```

Comment:
This comment is for the PIX capture helper, and I think it's still valuable information.